crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -142,14 +130,23 @@ impl ServerBuilder {
         self
     }
 
+    pub fn layer<L>(mut self, layer: L) -> Self
+    where
+        L: Layer<Route> + Clone + Send + 'static,
+        L::Service: Service<Request<Body>> + Clone + Send + 'static,
+        <L::Service as Service<Request<Body>>>::Response: IntoResponse + 'static,
+        <L::Service as Service<Request<Body>>>::Error: Into<Infallible> + 'static,
+        <L::Service as Service<Request<Body>>>::Future: Send + 'static,
```

Comment:
Out of interest, is it possible to write these without the `as` clauses?

```suggestion
        L::Service::Response: IntoResponse + 'static,
        L::Service::Error: Into<Infallible> + 'static,
        L::Service::Future: Send + 'static,
```

I know the LSP often writes things out fully explicitly, but often that can be avoided, so just curious.